packages/wonder-blocks-search-field/src/components/__tests__/search-field.test.js

```diff
@@ -285,4 +285,42 @@ describe("SearchField", () => {
         // Assert
         expect(clearButton).toHaveAttribute("aria-label", "test-clear-label");
     });
+
+    test("forwards the ref to the input element", () => {
+        // Arrange
+        const ref = React.createRef();
+
+        // Act
+        render(
+            <SearchField
+                id="sf-id"
+                value="some-value"
+                onChange={() => {}}
+                testId="search-field-test"
+                ref={ref}
+            />,
+        );
+
+        // Assert
+        expect(ref.current).toBeInstanceOf(HTMLInputElement);
+    });
+
+    test("forwards the ref to the input element with the expected value", () => {
+        // Arrange
+        const ref = React.createRef();
+
+        // Act
+        render(
+            <SearchField
+                id="sf-id"
+                value="some-value"
+                onChange={() => {}}
+                testId="search-field-test"
+                ref={ref}
+            />,
+        );
+
+        // Assert
+        expect(ref.current.value).toBe("some-value");
```

Comment:
I'm actually getting a flow error here saying `Cannot get `ref.current.value` because property `value` is missing in null [1].`.

Would the test still test correctly if I change it to `expect(ref.current?.value).toBe("some-value");`? Or would it pass when it shouldn't in that case?

Reply:
`expect(ref.current?.value).toBe("some-value");` should work just fine.

Reply:
In that case you can use `ref.current?.value` safely.

Reply:
I added `ref.current?.value` but Wallaby shows that line as yellow now. In a component file, I know yellow means that the line is only partially covered. What does yellow mean in a test file? Do I need to add an if/else and auto-fail in the else case?

Reply:
The following is green on wallaby and passes flow:

```expect(ref.current && ref.current.value).toBe("some-value");```

Should I just keep it like this instead of using `?`?

Reply:
mmm I'm not sure what yellow means in the test context.... @kevinbarabash maybe do you have any thoughts on that?